.github/workflows/pr-e2e.yml

```diff
@@ -44,6 +42,16 @@ jobs:
     container: ghcr.io/kedacore/build-tools:main
     if: needs.triage.outputs.run-e2e == '1'
     steps:
+      - uses: actions/github-script@v6
+        name: Remove ok-to-merge label
+        with:
+          script: |
+            github.rest.issues.removeAllLabels({
```

Comment:
Remove all labels or just `tests/*`?

Reply:
All labels.
I used this behavior because I get an error (404) when I try to delete a tag that doesn't exist. 
I can delete all or list the tags before to know if the e2e tag is there and then delete it or not. The point is that yesterday I reviewed merged PRs and we don't labeled them, so I assumed that deleting all is doable, but if you don't agree, I can update the PR for listing labels, looking for our label, and then removing only it.

Reply:
We don't have them yet but sometimes there are some or we might add them in the future.

I think for this PR it's OK but should do a follow-up/create issue to only delete the one we want

Reply:
I can try to improve getting the current labels before and deleting only if the label is there

Reply:
I have discovered that gh-cli solves this problem, so I have updated the PR